frontend/src/shared/components/baseCard/baseCard.css

```diff
@@ -87,21 +87,28 @@
 }
 
 .card-header {
-    height: 80px;
+    height: 16vw;
     background-size: 100%;
     background-position-x: center;
     background-repeat: no-repeat;
 }
 
-@media (max-width: 768px) {
+/*As more columns show up the vw calculation needs to be specifically adjusted*/
+@media (min-width: 1024px) {
     .card-header {
-        height: 25vh;
+        height: 8vw;
```

Comment:
Did you mean vh? not vw.

Also I think vh is what is causing all this headache over resizing and stuff. May be better to just set fixed widths and heights for cards at different resolutions and let the tiling take care of the rest

Reply:
Originally what's what I also thought, but vw is actually a much better predictor for the number of columns and the height of the cards. If we use vh, and do some calculation, based off the card element size, maybe we can get closer? I'm not sure what the more elegant solution is.

Reply:
yeah, I think having hard px values is better if we have a fixed same for the header and we must take into account the width of the base card also changed based on the resolution. The solution I can think of is
`
.card-header {
  width: 100% 
}
`
Then we can have a forced rule that all headers must match a resolution aspect (4:3 or something)

Reply:
I agree with Jim. Enforcing the aspect ratio is probably the best fix here.